pychromecast/controllers/media.py

```diff
@@ -613,6 +613,11 @@ def _send_start_play_media(  # pylint: disable=too-many-locals
                 "edgeColor": "#000000FF",
             }
 
+        # if user has specified metadata but not specified metadataType
+        # https://developers.google.com/cast/docs/reference/messages#MediaInformation
+        if media["metadata"].keys() and "metadataType" not in media["metadata"]:
```

Comment:
Why do we need to check if metadata's keys is truthy, isn't this enough:
```suggestion
        if "metadataType" not in media["metadata"]:
```

Reply:
because metadata is initialized as an empty dict. We only want to set the MetadataType if the user has defined other metadata attributes, unless that is counter to the intention.

Reply:
OK, got it.
An empty dict is falsy, there's no need to check the keys, I updated the suggestion accordingly.